test/staking/StakeManager.test.js

```diff
@@ -1014,6 +1067,7 @@ contract('StakeManager:validator replacement', async function (accounts) {
       assert.ok(await stakeManager.isValidator(logs[4].args.newValidatorId))
     })
 
+    // remove this?
     it('should confrim auction and secure the place for validator itself', async function () {
       // let validator = await stakeManager.validators(2)
       // let stake = validator.amount
```

Comment:
@0xAshish Do we need this test?

Reply:
yup please remove will add this test in different PR.